### DISCUSSION TITLE
Meetup'y w Warszawie

### DISCUSSION THREAD

**Discussion by magliarosa** [AzureCommunityPL/group#476] [Wydarzenia]
Cześć,

Kiedy możemy spodziewać się kolejnych meetupów w Warszawie?
Wiem, że nasz host zmieniał siedzibę w styczniu, ale czy w lutym jest szansa na kolejne spotkanie?

**Comment by eklime** [comment]
To jest bardzo dobre pytanie. Znamy nawet na nie odpowiedź - w lutym spotkania nie będzie.
Wyjaśnienie jest dość trywialne - brak chętnych do wygłoszenia sesji.

I to nie jest tak, że byśmy prelegentów nie znaleźli, ale już kiedyś dostaliśmy plaskacza w twarz, że występują ciągle "te same osoby". Nikt tylko nie pochylił się nad faktem, że to nie wina organizatora, a braku chętnych, pomimo że jako organizatorzy oferujemy wsparcie w przygotowaniu prelekcji.

Biorąc powyższe pod uwagę, jak już napisałem w pierwszym zdaniu - w lutym, podobnie jak w styczniu spotkania nie będzie.

Najprawdopodobniej zorganizujemy coś w marcu. Jeśli nikt się nie zgłosi, to wystąpi "ta sama ekipa" aby nie przepadła nam kolejna okazja do spotkania.

P.S. Jeśli ktoś chciałby wystąpić na spotkaniu w Warszawie (niekoniecznie już na najbliższym spotkaniu) to zapraszam do kontaktu. Można wrzucić maila na kontakt@azurecommunity.pl 

Pozdro,
Emil

CC: @mgrabarz  @kaluzaaa  @wi5nia  @smereczynski

**Comment by mgrabarz** [comment]
Potwierdzam. Nie będzie chętnych, to nie będzie meetupow. 

Chętnie przyjmę tez kogoś do organizacji :)
